src/App.jsx

```diff
@@ -14,57 +16,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectProduct, setSelectProduct] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleSelectGoods = (good) => {
+    setSelectProduct(good === selectProduct ? null : good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const GoodsList = ({ goodsList }) =>
```

Comment:
Move the component outside the `App` component. Each component should have a separate file

Reply:
Write everything inside the App (don't create additional components).

Reply:
You should keep everything inside the App component as required, not to create additional components. However, if you do create a `GoodsList `component, it's advisable to move it to a separate file. This separation enhances code organization and readability,